Tests/HaCWebsiteLibTests/HaCWebsiteLibTests.swift

```diff
@@ -2,16 +2,94 @@ import XCTest
 @testable import HaCWebsiteLib
 
 class HaCWebsiteLibTests: XCTestCase {
```

Comment:
It seems to me like this file isn't testing `HacWebsiteLib` but just `Workshop`. Should the name of the file reflect that?